src/test/java/bio/terra/service/auth/iam/sam/SamApiServiceTest.java

```diff
@@ -38,7 +44,14 @@ public class SamApiServiceTest {
   void setUp() {
     when(configurationService.getParameterValue(ConfigEnum.SAM_OPERATION_TIMEOUT_SECONDS))
         .thenReturn(OPERATION_TIMEOUT_SECONDS);
-    samApiService = new SamApiService(samConfig, configurationService);
+    SdkTracerProvider sdkTracerProvider =
+        SdkTracerProvider.builder()
+            .addSpanProcessor(SimpleSpanProcessor.create(LoggingSpanExporter.create()))
+            .build();
+    var traceProvider =
+        OpenTelemetrySdk.builder().setTracerProvider(sdkTracerProvider).build().getTracerProvider();
+    when(openTelemetry.getTracerProvider()).thenReturn(traceProvider);
```

Comment:
Is there a reason that this mock set-up is so extensive here?  I think other services may disable OTEL for unit tests in properties.

Reply:
Yeah, this was not ideal. Thanks for pointing that out -- following [BPM's example](https://github.com/DataBiosphere/terra-billing-profile-manager/commit/f94c66300b72262974bbde44ba8768780a8187cb#diff-cec3b0c74c24475c0cd84bfa7fed973efcd0ca4d4597b9d48c14f55696752b92R59), we can just call "OpenTelemetry.noop()" when initializing the sam service in the test.